src/AuditTrailModule/javasource/test_crm/tests/TestAuditInheritance.java

```diff
@@ -0,0 +1,226 @@
+package test_crm.tests;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Date;
+
+import com.mendix.core.Core;
+import com.mendix.core.CoreException;
+import com.mendix.systemwideinterfaces.core.IContext;
+import com.mendix.systemwideinterfaces.core.IMendixIdentifier;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import audittrail.proxies.TypeOfLog;
+import system.proxies.User;
+import test_crm.proxies.Company;
+import test_crm.proxies.Group;
+import test_crm.tests.actual.ActualLog;
+import test_crm.tests.expected.ExpectedLogLineAttribute;
+import test_crm.tests.expected.ExpectedLog;
+import test_crm.tests.expected.ExpectedLogLineReferenceSet;
+
+public class TestAuditInheritance {
+	private IContext context;
+
+	private IMendixIdentifier adminId;
+	private Group group;
+	private Group group2;
+
+	@Before
+	public void beforeTesting() throws CoreException {
+		this.context = Core.createSystemContext();
+
+		this.adminId = User.load(this.context, String.format("[%1s = '%2s']", User.MemberNames.Name, Core.getConfiguration().getAdminUserName()))
+			.get(0).getMendixObject().getId();
+
+		this.group = new Group(context);
```

Comment:
Originally, because the method was so simple that there was very little to gain in clarity. But then I moved it all to a separate class for funsies and I added this method as well.